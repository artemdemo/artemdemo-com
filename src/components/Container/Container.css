.container__ {
  max-width: 900px;
}
