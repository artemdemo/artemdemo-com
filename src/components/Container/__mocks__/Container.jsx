import React from 'react';
import _omit from 'lodash/omit'

const Container = props => (
    <div data-mock='Container'>
        {JSON.stringify(_omit(props, 'children'), null, 2)}
        {props.children}
    </div>
);

export default Container;