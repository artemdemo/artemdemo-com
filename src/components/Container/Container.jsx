import React from 'react';
import classnames from 'classnames';
import './Container.css';

const Container = ({ children, className }) => (
  <div className={classnames(className, 'container container__')}>
    {children}
  </div>
);

export default Container;
