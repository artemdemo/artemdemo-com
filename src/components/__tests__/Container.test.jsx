import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../Container';

describe('Container', () => {
    it('should render empty', () => {
        const tree = renderer.create(
            <Container />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('should render with child', () => {
        const tree = renderer.create(
            <Container>
                <div>Some child</div>
            </Container>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});