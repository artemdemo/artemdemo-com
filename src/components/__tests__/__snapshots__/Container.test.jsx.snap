// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render empty 1`] = `
<div
  className="sc-bdVaJa AOLhf"
/>
`;

exports[`Container should render with child 1`] = `
<div
  className="sc-bdVaJa AOLhf"
>
  <div>
    Some child
  </div>
</div>
`;
